Aptoide/aptoide-client-v8
# [feature/MOB-785 & MOB-784] Search Filters

**What does this PR do?**

   This PR Implements the new search filters. This also includes thew new empty state, MOB-784.

**Database changed?**

   No

**How should this be manually tested?**

  Check general search and specific store search. Both should work correctly and as expected. Test the different the filters (and different combinations) and check if they're working as expected (also in older devices). 

**What are the relevant tickets?**

  [MOB-785](https://aptoide.atlassian.net/browse/MOB-785)
  [MOB-784](https://aptoide.atlassian.net/browse/MOB-784)



**Code Review Checklist**
